### PR TITLE
mark totalInterFrameDelay and totalSquaredInterFrameDelay as video-only

### LINKED ISSUE #656: totalInterFrameDelay, and totalSquaredInterFrameDelay are video only
totalInterFrameDelay, totalSquaredInterFrameDelay are not marked with "Only [exist](https://infra.spec.whatwg.org/#map-exists)s for video."

### PR DESCRIPTION
**Pull request by fippo**
fixes #656

***
<a href="https://pr-preview.s3.amazonaws.com/fippo/webrtc-stats/pull/733.html" title="Last updated on Mar 7, 2023, 2:28 PM UTC (1db3085)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-stats/733/2ada0c9...fippo:1db3085.html" title="Last updated on Mar 7, 2023, 2:28 PM UTC (1db3085)">Diff</a>

### CODE REVIEW

**Review by henbos — APPROVED** [review]

File: webrtc-stats.html [hunk 1]
```diff
@@ -1190,6 +1190,7 @@ <h2>
               </dt>
               <dd>
                 <p>
+                  Only [= map/exist =]s for video.
```
Comment (vr000m) [inline webrtc-stats.html:1193]: Should this be consistent with "Does not [= map/exist =] for audio."

**Review by vr000m — APPROVED** [review]
make consistent with the language in the other PR?

File: webrtc-stats.html (hunk 1 above)
Comment (henbos) [inline webrtc-stats.html:1193]:
```suggestion
                  Does not [= map/exist =] for audio.
```

File: webrtc-stats.html [hunk 2]
```diff
@@ -1204,6 +1205,7 @@ <h2>
...
```
Comment (vr000m) [inline webrtc-stats.html:1208]:
```suggestion
                  Does not [= map/exist =] for audio.
```

File: webrtc-stats.html (hunk 1 above)
Comment (henbos) [reply webrtc-stats.html:1193]: I'll commit a change

**Pull request merged**